test/render.js

```diff
@@ -254,8 +252,8 @@ describe('render', () => {
 				.to.have.been.calledOnce
 				.and.calledWithExactly(
 					match(PROPS),
-					match({}),	// empty state
```

Comment:
with preact 10,  state can now null ?

Reply:
yup, we aligned our behaviour with `react`'s which initializes it with `null`.